projectroles/management/commands/cleanappsettings.py

```diff
@@ -16,6 +16,7 @@
 ALLOWED_TYPES_MSG = 'does not match allowed types'
 DELETE_PREFIX_MSG = 'Deleting "{}" from project "{}": '
 DELETE_PROJECT_TYPE_MSG = 'project type "{}" {}: {}'
+DELETE_SCOPE_MSG = 'user {} has no Role in the project'
```

Comment:
For logging conventions we can simplify this into `user {} has no role in project`.